projects/plugins/jetpack/extensions/blocks/conversation/edit.js

```diff
@@ -52,28 +37,56 @@ function ConversationEdit( { className, attributes, setAttributes } ) {
 						...updatedParticipant,
 					};
 				} ),
-			} ),
+			} );
+		},
 		[ setAttributes, participants ]
 	);
 
+	const addNewParticipant = useCallback( function( newSpakerValue ) {
```

Comment:
> * We've already discussed the positioning of the autocomplete, but want to flag that again here. When the autocomplete appears above the text entry rather than below, the placement feels unexpected based on existing autocomplete patterns.

Not sure what do you exactly mean with  _existing autocomplete patterns_. I think all autcompleters in core behave in the same way. It's a popover with a [default `top right` position](https://github.com/WordPress/gutenberg/blob/d8d208927e964bf4f928a9e6b378902ed5fc421e/packages/components/src/autocomplete/index.js#L244).
**Is it a blocker?**
Btw, [I created a PR](https://github.com/WordPress/gutenberg/pull/28611) that allows extending the Popover properties, in the same that other components do, like the Dropdown component. It won't be an immediate solution but if it's merged in the core, it will be updated in the near future.

> It also obstructs the view of the UI in the toolbar.

The autocomplete always overlaps over the canvas area. It happens for all of them. Typing `/` in a paragraph block, for instance.

<img src="https://user-images.githubusercontent.com/77539/106644339-bf46ed00-6569-11eb-83ce-e7578f966f51.png" width="400px" />

Even more, if you press `TAB` it won't be possible to hide the blocks selector via keyboard. In that case, you need to click on another canvas area.

I've updated this PR forcing to hide the autocomplete when the component loses the focus, via the `TAB` key for instance. The slightly bad news is the `onFocusOutside` event isn't available in the current Gutenberg version for Jetpack. It will work in other sites with newer core versions.

> If I hadn't seen your previous screen recordings, I wouldn't know I could edit the speaker label text styling. (see screenshot below)

I think it's a matter to hide the autocomplete. Just selects the participant and the popover will disappear or pressing the `ESC` key. The previous comment above will avoid this situation, too.

> I wonder if we could make the speaker label text bold by default to distinguish it from the transcription text and move the label color settings to the sidebar so that it doesn't get lost.

Trying....

Reply:
https://github.com/Automattic/jetpack/pull/18646/commits/7de2e0b9f9775e906b5409f795d532378a29ddd6